packages/web/src/common/store/notifications/sagas.ts

```diff
@@ -153,6 +158,22 @@ export function* fetchNotifications(action: FetchNotifications) {
         withDethroned
       })
     )
+
+    if (useDiscoveryNotifications && 'notifications' in notificationsResponse) {
+      const discoveryNotifications = yield* call(() => {
```

Comment:
kinda a nit, but i think you can just create all the variables outside the call, and then just do `call(audiusBackendInstance.getDiscoveryNotifications, ...)`